libr/core/linux_heap_glibc.c

```diff
@@ -849,63 +853,123 @@ void GH(print_heap_fastbin)(RCore *core, GHT m_arena, MallocState *main_arena, G
 	}
 }
 
-static void GH(print_tcache_instance)(RCore *core, GHT m_arena, MallocState *main_arena) {
-	if (!core || !core->dbg || !core->dbg->maps) {
-		return;
+static GH(RTcache)* GH(tcache_new) (tcache_type type) {
+	GH(RTcache) *tcache = R_NEW0 (GH(RTcache));
```

Comment:
add a space before every (

Reply:
In the code there are `GH(` (which is more readable IMHO) instead of `GH (`, should I change them all?

Reply:
I also think `GH(` should have no space and it is more readable... We also don't put space in `Elf_` in the elf code, which uses a similar structure.